app/controllers/users_controller.rb

```diff
@@ -38,7 +38,11 @@ def get_user
   end
 
   def index
-    @users = User.limit(6).page(params[:page])
+    if params[:query]
+      @users = User.search(params[:query]).limit(6).page(params[:page])
+    else
+      @users = User.order('created_at DESC').limit(6).page(params[:page])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.